01-deep-dive.md

```diff
@@ -152,6 +152,33 @@ $('#summernote').summernote({
 });
 {% endhighlight %}
 
+### Custom styles
+
+You can set your own selection of styles with the `styleTags` option.
+
+{% highlight javascript %}
+$('#summernote').summernote({
+  styleTags: [
+    'p',
+        { title: 'Blockquote', tag: 'blockquote', className: 'blockquote', value: 'blockquote' },
+        'pre', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6'
+	],
+  });
+{% endhighlight %}
+
+The tags can be specified just by tag name (as with `p` or `pre` or `h1`-`h6` above). It is also
+possible to customize the style in more defaul by providing an object looking like:
```

Comment:
You have a typo "defaul" == "default".
Good work on this PR, I'll tag the others so they can review it for merging.

Reply:
My bad, got it. :) It was actually "defaul" -> "detail". Thanks @DiemenDesign !